protos/remote_id/remote_id.proto

```diff
@@ -0,0 +1,294 @@
+        HEIGHT_REF_OVER_GROUND = 1; // The height field is relative to ground.
+    }
+
+    Status status = 1; // Indicates whether the unmanned aircraft is on the ground or in the air.
+    uint32 direction_deg = 2; // Direction over ground measured clockwise from true North: 0 - 359 deg.
+    float speed_horizontal_m_s = 3; // Ground speed in meters per second. Positive only.
+    float speed_vertical_m_s = 4; // The vertical speed in meters per second. Up is positive.
+    double latitude_deg = 5; // Current latitude of the unmanned aircraft
+    double longitude_deg = 6; // Current longitude of the unmanned aircraft
+    float altitude_barometric_m = 7; // The altitude calculated from the barometric pressure.
+    float altitude_geodetic_m = 8; // The geodetic altitude as defined by WGS84.
+    HeightRef height_reference = 9; // Indicates the reference point for the height field.
+    float height_m = 10; // The current height of the unmanned aircraft. As indicated by height_reference.
+    uint64 time_utc_us = 11; // Timestamp in UTC (since UNIX epoch) in microseconds
+}
+
+message LocationAccuracy {
+    enum HorAcc {
+        HOR_ACC_UNKNOWN = 0; // The horizontal accuracy is unknown.
+        HOR_ACC_NM_10 = 1; // The horizontal accuracy is smaller than 10 Nautical Miles. 18.52 km.
+        HOR_ACC_NM_4 = 2; // The horizontal accuracy is smaller than 4 Nautical Miles. 7.408 km.
+        HOR_ACC_NM_2 = 3; // The horizontal accuracy is smaller than 2 Nautical Miles. 3.704 km.
+        HOR_ACC_NM_1 = 4; // The horizontal accuracy is smaller than 1 Nautical Miles. 1.852 km.
+        HOR_ACC_NM_0_5 = 5; // The horizontal accuracy is smaller than 0.5 Nautical Miles. 926 m.
+        HOR_ACC_NM_0_3 = 6; // The horizontal accuracy is smaller than 0.3 Nautical Miles. 555.6 m.
+        HOR_ACC_NM_0_1 = 7; // The horizontal accuracy is smaller than 0.1 Nautical Miles. 185.2 m.
+        HOR_ACC_NM_0_05 = 8; // The horizontal accuracy is smaller than 0.05 Nautical Miles. 92.6 m.
+        HOR_ACC_METER_30 = 9; // The horizontal accuracy is smaller than 30 meter.
+        HOR_ACC_METER_10 = 10; // The horizontal accuracy is smaller than 10 meter.
+        HOR_ACC_METER_3 = 11; // The horizontal accuracy is smaller than 3 meter.
+        HOR_ACC_METER_1 = 12; // The horizontal accuracy is smaller than 1 meter
+    }
+
+    enum VerAcc {
+        VER_ACC_UNKNOWN = 0; // The vertical accuracy is unknown.
+        VER_ACC_METER_150 = 1; // The vertical accuracy is smaller than 150 meter.
+        VER_ACC_METER_45 = 2; // The vertical accuracy is smaller than 45 meter.
+        VER_ACC_METER_25 = 3; // The vertical accuracy is smaller than 25 meter.
+        VER_ACC_METER_10 = 4; // The vertical accuracy is smaller than 10 meter.
+        VER_ACC_METER_3 = 5; // The vertical accuracy is smaller than 3 meter.
+        VER_ACC_METER_1 = 6; // The vertical accuracy is smaller than 1 meter.
+    }
```

Comment:
Still puzzles my why they wouldn't just use a `uint8` to signal meter accuracy... but anyway, I think I can live with that.

Reply:
I think the reason is historic. In some radio broadcasting ID protocols (like AIS or ADSB) the accuracy is an enumerative type or is rounded to less than 8 bits to save space (then save bandwidth)

Reply:
Right, fair enough.